### PR TITLE
HW2

### PR DESCRIPTION
**Pull request by ritakuo**
HW2 dated 10/12/2015

### CODE REVIEW

File: rita/HW2/css/main.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.feature{
+margin:20px, 40px, 20px, 40px;
+height:300px
+}
+
+
+.contact{
+	text-align:right;
+}
+#name{
```
Comment (techbow) [inline rita/HW2/css/main.css:33]: usually we use class selector rather than id selector for css

File: rita/HW2/css/main.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	border-radius:100px 100px 100px 100px;
+	padding:20px;
+}
+
+.header{
+margin:20px, 0, 40px, 0
+}
+
+.feature{
+margin:20px, 40px, 20px, 40px;
```
Comment (techbow) [inline rita/HW2/css/main.css:25]: css shortcut margin: 20px, 40px

File: rita/HW2/css/main.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+}
+
+/* This is for the photo */
+#photo{
+	width:150px;
+	height:150px;
+	float:left;
+	margin-left:10px;
+	border-radius:100px 100px 100px 100px;
```
Comment (techbow) [inline rita/HW2/css/main.css:16]: you can short it border-radius: 100px;

File: rita/HW2/css/main.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
+
+/* This is for the default value */
+body {
+	font-family: Verdana;
```
Comment (techbow) [inline rita/HW2/css/main.css:4]: usually we have at least 3 font family and last one can be serif or sans serif

File: rita/HW2/css/main.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+	width:150px;
+	height:150px;
+	float:left;
+	margin-left:10px;
+	border-radius:100px 100px 100px 100px;
+	padding:20px;
+}
+
+.header{
+margin:20px, 0, 40px, 0
```
Comment (techbow) [inline rita/HW2/css/main.css:21]: incention

File: rita/HW2/resume.html [hunk 6]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title> Rita's resume</title>
+	<link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+	<div class="header">
```
Comment (techbow) [inline rita/HW2/resume.html:9]: you can use semantic tag header here

File: rita/HW2/css/main.css (hunk 5 above)
Comment (ritakuo) [inline rita/HW2/css/main.css:21]: what is incention?

**Pull request merged**